food.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
don't mix single quotes a double quotes. Also, place your `"use strict";` statement inside the IIFE.

```js
var Food = (function(food) {
  "use strict";

  var targetElement = document.getElementById("food-table-body");
```